cou/cli.py

```diff
@@ -247,6 +242,9 @@ def entrypoint() -> None:
             "$ COU_TIMEOUT=60 cou plan"
         )
         sys.exit(1)
+    except CloudVerificationError as exc:
+        logger.error(exc)
+        sys.exit(1)
```

Comment:
Can we reuse the COUException handler below, to avoid adding more code?

Reply:
Yes, we can refactor the UI later. `COUException` will also print out `"See the known issues at https://canonical-charmed-openstack-upgrader.readthedocs-....` which is not true for our case: we can fix the issue; it's no a known issue